client/dist/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Lodge.io Owner Description</title>
     <link rel="stylesheet" type="text/css" href="./style.css">
```

Comment:
@SimonC989 @carl12 CSS Modules or React Styled Components should be used to add CSS so as to avoid styling collisions between group members' components.